helm/ffc-ahwr-application/values.yaml

```diff
@@ -33,6 +33,8 @@ deployment:
   replicas: 1
 
 container:
+  backOfficeRequestQueueAddress: ffc-ahwr-backoffice-request
```

Comment:
Can thees keys be added in alphabetical order please, makes it easier to scan through.